Source/JavaScriptCore/b3/B3LowerToAir.cpp

```diff
@@ -4469,6 +4469,25 @@ class LowerToAir {
         }
 
         case Select: {
+            if (m_value->type().isVector()) {
+                // Conditional moves aren't widely supported for vectors, so
+                // we lower vector Select to a branching construct.
```

Comment:
Since current architectures do not support conditional moves for vectors, can you change this comment to be more clear?  As is, it implies that some arch do support it but we're just going with the lowest common denominator here.  How about something like, "Conditional moves are available for vectors on currently supported architectures. So, we lower ..."?

Reply:
Oops, I meant "Conditional moves are **not** available ..."